client/src/components/AvatarDOM.tsx

```diff
@@ -32,64 +44,31 @@ function AvatarDOM({
     }
   }, []);
 
+  function renderStatusIcon() {
+    if (!active) {
+      return <Icon className="active-icon" name="RemoveSolid" color="danger700" size="1.5rem" />;
+    } else if (mute) {
+      return <MicOffIcon />;
+    }
+
+    return null;
+  }
+
   return (
     <div
-      className="avatar"
+      className={cx("avatar", "avatar-outer")}
       onPointerDown={onPointerDown}
-      style={{
-        touchAction: "none",
-        width: "4.5rem",
-        height: "4.5rem",
-        borderRadius: "100%",
-        boxShadow: "0 0 0 " + (1 + multiplier * 10).toString() + "px " + _borderColor,
-        background: _backgroundColor,
-        position: "absolute",
-        display: "table",
-        left: pos[0],
-        top: pos[1],
-      }}
+      style={calculateSpecificStyles()}
     >
       <div
-        style={{
-          userSelect: "none",
-          color: "white",
-          verticalAlign: "middle",
-          textAlign: "center",
-          display: "table-cell",
-          fontFamily: "helvetica",
-          fontSize: "2rem",
-          cursor: "default",
-        }}
+        className="avatar-initial"
```

Comment:
This should probably be one line. I think Prettier should be able to just handle it?

Reply:
Yeah agreed